.gitignore

```diff
@@ -54,7 +55,7 @@ tramp
 .idea/
 .phpstorm.meta.php
 # NetBeans
-nbproject/
+nbproject/private
```

Comment:
Should we be IDE agnostic?

Reply:
Yes, this sub folder contains my local settings for IDE and this is the reason why it was ignored. Everithing else contains the SuiteCRM specific changes, in our case the Unit Test settings and required PHP version for SuiteCRM etc. This change was for adding the ability to users to run test and code coverage whit netbeans if they wanted (see above in the PR description). - Already removed.